multirpc/subpub/gossipsub.go

```diff
@@ -0,0 +1,103 @@
+package subpub
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"go.vocdoni.io/dvote/log"
+)
+
+// ChatRoomBufSize is the number of incoming messages to buffer for each topic.
+const ChatRoomBufSize = 128
+
+// ChatRoom represents a subscription to a single PubSub topic. Messages
```

Comment:
i kept the "ChatRoom" name although i'd love a more... neutral name. I mean, nodes are not chatting.
maybe "Broadcast" would do.
it sounds nice, would have channel Broadcast.Messages
and method Broadcast.Publish, and so on.

Reply:
i opted for "Gossip"